### ISSUE TITLE
Preview tab does not update for previously unrectified map

### ISSUE THREAD

**Issue by lklepner** [nypl-spacetime/nypl-warper#34]
I've noticed that if you view the preview tab of an unrectified map it displays the message "Sorry, you'll have to rectify a map before you can view it." and then you rectify the map and return to the preview tab the unrectified message remains - a page refresh is needed to access the preview tab at that point.

**Comment by mgiraldo** [comment]
yeah. work in one tab should "refresh" all others that could be affected.
